daemon/api.go

```diff
@@ -622,6 +622,7 @@ func searchStore(c *Command, r *http.Request, user *auth.UserState) Response {
 	q := query.Get("q")
 	section := query.Get("section")
 	name := query.Get("name")
+	scope := query.Get("scope")
```

Comment:
should we check whether scope is only "wide" or empty ?

Reply:
or from another POV, do we get a sane enough error if it's set to something else?

Reply:
I see no harm in passing it through, as long as it's not exposed to the end-user directly